data/category.go

```diff
@@ -169,7 +183,7 @@ func (s *specification) Test(cfg write.ClientConfig) error {
 	}
 
 	if !eq {
-		return fmt.Errorf("expected:\n%s\vgot:\n%s\n", exp, got)
+		return fmt.Errorf("Query: %v\nExpected: \n%s\vGot: \n%s\n", string(q), exp, got)
```

Comment:
This seems fine for now, but eventually we should have something that produces a more intelligent, JSON-aware diff. I don't know of any such library off the top of my head.